dask_jobqueue/pbs.py

```diff
@@ -85,13 +85,15 @@ def __init__(self, queue=None, project=None, resource_spec=None, walltime=None,
             header_lines.append('#PBS -A %s' % project)
         if resource_spec is None:
             # Compute default resources specifications
-            ncpus = self.worker_processes * self.worker_threads
-            memory = self.worker_memory * self.worker_processes
-            memory_string = pbs_format_bytes_ceil(memory)
-            resource_spec = "select=1:ncpus=%d:mem=%s" % (ncpus, memory_string)
+            resource_spec = "select=1:ncpus=%d" % self.worker_cores
+            if self.worker_memory:
+                memory = self.worker_memory * self.worker_processes
```

Comment:
Same as `self.worker_cores`, I propose to just record and use a `self.worker_total_memory`, or `self.memory`. 
Except if you've done that because of the conversion from string to int and the other way around.

Reply:
Ah indeed.  That was an oversight.  I've now just changed the definition of worker_memory to mean total memory.